addons/github/tests/utils.py

```diff
@@ -130,18 +138,304 @@ def create_mock_github(user='octo-cat', private=False):
      u'watchers_count': 1469,
      # NOTE: permissions are only available if authorized on the repo
      'permissions': { 'push': True }
-     }))
-
-    github_mock.branches.return_value = [
-        Branch.from_json(dumps({u'commit': {u'sha': u'e22d92d5d90bb8f9695e9a5e2e2311a5c1997230',
-           u'url': u'https://api.github.com/repos/{user}/mock-repo/commits/e22d92d5d90bb8f9695e9a5e2e2311a5c1997230'.format(user=user)},
-          u'name': u'dev'})),
-         Branch.from_json(dumps({u'commit': {u'sha': u'444a74d0d90a4aea744dacb31a14f87b5c30759c',
-           u'url': u'https://api.github.com/repos/{user}/mock-repo/commits/444a74d0d90a4aea744dacb31a14f87b5c30759c'.format(user=user)},
-          u'name': u'master'})),
-         Branch.from_json(dumps({u'commit': {u'sha': u'c6eaaf6708561c3d4439c0c8dd99c2e33525b1e6',
-           u'url': u'https://api.github.com/repos/{user}/mock-repo/commits/c6eaaf6708561c3d4439c0c8dd99c2e33525b1e6'.format(user=user)},
-          u'name': u'no-bundle'}))
-      ]
+     }), session)
+
+
+
+    branch_1 = Branch.from_json(dumps({
+        u'commit': {
```

Comment:
If we could phase out the u-strings for py3 that would be great. We should have a syntax moot some day and decide these details. [The deal with u-strings in py3](https://stackoverflow.com/questions/2464959/whats-the-u-prefix-in-a-python-string)